app/components/publishing-log-hash.js

```diff
@@ -0,0 +1,16 @@
+import Component from '@glimmer/component';
+import { tracked } from '@glimmer/tracking';
+import { task } from 'ember-concurrency';
+
+export default class PublishingLogHashComponent extends Component {
+  @tracked hash;
+
+  @task
+  *loadData() {
```

Comment:
This breaks the fundamental ember contract of recalculating dependent data when their reactive dependencies change. This function depends on `this.args.log`, but it will not recalculate when that changes.

this is exactly _why_ ember-resources exist. 

Same thing in publishing-log-document-name